docs/developer/api/rooms.rst

```diff
@@ -39,3 +39,17 @@ Allowed reactions currently are:
 * ``+1``
 * ``open_mouth``
 * ``heart``
+
+Schedule changes
+----------------
+
+Moderators can update the current ``schedule_data`` field like this::
+
+    => ["room.schedule", 123, {"room": "room_1", "schedule_data": {"session": 1}}]
+    <- ["success", 123, {}]
+
+Permitted session data keys are ``session`` and ``title``.
+
+When the schedule data is updated, a broadcast to all users in the room is sent::
```

Comment:
Just now noticed the `to all users in the room`.
The frontend uses the schedule data in the new dashboard and the sidebar, so the broadcast is actually needed for all users worldwide.
Sorry that I just noticed that now.